src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/IHtmlFragmentCache.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    public interface IHtmlFragmentCache
+    {
+        Task<IHtmlContent> SetAsync(object key, Func<Task<IHtmlContent>> renderContent, HtmlFragmentCacheContext context);
```

Comment:
Should the key be a `string`? The Key property and the auto generated key on the cache tag helper are strings. Also the key on IDistributedCache is a string.